dist/common/scripts/scylla_util.py

```diff
@@ -367,7 +367,7 @@ def is_gentoo_variant():
     return ('gentoo' in os_release['ID'])
 
 def redhat_version():
-    return int(os_release['VERSION_ID'])
+    return float(os_release['VERSION_ID'])
 
```

Comment:
It's not good practice to compare floats, rounding errors can cause confusion.

Let's return os_release['VERSION_ID'].split('.'), then we can check individual components specifically (e.g. redhat_version()[0])

Reply:
> It's not good practice to compare floats, rounding errors can cause confusion.
> 
> Let's return os_release['VERSION_ID'].split('.'), then we can check individual components specifically (e.g. redhat_version()[0])

Then we need to convert each value to int before compare, and we can't guarantee the item number of return list.

Currently we only touched number and '.' in VERSION_ID, but it might contains '_' or '-'.
It's robust to compare version string by pkg_resources.parse_version().

Quote from: https://www.linux.org/docs/man5/os-release.html

```
       VERSION_ID=
           A lower-case string (mostly numeric, no spaces or other characters outside of 0-9, a-z, ".", "_" and "-")
           identifying the operating system version, excluding any OS name information or release code name, and
           suitable for processing by scripts or usage in generated filenames. This field is optional. Example:
           "VERSION_ID=17" or "VERSION_ID=11.04".
```